docs/guides/postgres/audit-logging/pgaudit.md

```
@@ -0,0 +1,653 @@
+--------------------+------------
+ credcheck          | 5.0.0
+ pg_cron            | 1.6
+ pg_stat_statements | 1.10
+ pgaudit            | 16.1
+```
+
+Run this on the **primary**; a standby rejects it with
+`cannot execute CREATE EXTENSION in a read-only transaction`:
+
+```bash
+$ kubectl get pod -n bank -l kubedb.com/role=primary -o name
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)